docs/design/video-template.ep

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0"?>
+<Document xmlns="http://www.evolus.vn/Namespace/Pencil"><Properties/><Pages><Page><Properties><Property name="name">Untitled Page</Property><Property name="id">1427954692724_9831</Property><Property name="width">1127</Property><Property name="height">1412</Property><Property name="dimBackground"/><Property name="transparentBackground"/><Property name="backgroundColor">#ffffff</Property><Property name="fid">untitled_page</Property></Properties><Content><g xmlns="http://www.w3.org/2000/svg" p:type="Shape" xmlns:p="http://www.evolus.vn/Namespace/Pencil" p:def="Evolus.Common:RoundedRect" id="13deec86e9cd42029c89058c08e100a3" transform="matrix(1,0,0,1,121,122)" p:sizing-ox="0" p:sizing-oy="0" p:sizing-ow="730" p:sizing-oh="455"><p:metadata><p:property name="box"><![CDATA[705,444]]></p:property><p:property name="withBlur"><![CDATA[false]]></p:property><p:property name="radius"><![CDATA[0,0]]></p:property><p:property name="textPadding"><![CDATA[0,55.5]]></p:property><p:property name="fillColor"><![CDATA[#FFFFFFFF]]></p:property><p:property name="strokeColor"><![CDATA[#1B3280FF]]></p:property><p:property name="strokeStyle"><![CDATA[2|]]></p:property><p:property name="textContent"><![CDATA[
```

Comment:
Also considered elements alternatives where possible (ex. subtitles can be alternatively shown on the video)